src/styles/global.css

```diff
@@ -1770,3 +1770,109 @@ li.squeak-left-border {
 body.dark .max-ai-body {
     color: #151515;
 }
+
+@keyframes quest-log-sprite-walk {
+    0% {
+        background-position: 0 0;
+    }
+    100% {
+        background-position: -528px 0;
+    }
+}
+
+.quest-log-sprite-animate {
+    animation: quest-log-sprite-walk 0.25s steps(11) 7;
+}
+
+@keyframes speechBounce {
+    0% {
+        transform: translateY(-50%) scale(0.8) translateX(-10px);
+        opacity: 0;
+    }
+    50% {
+        transform: translateY(-50%) scale(1.05) translateX(2px);
+        opacity: 0.8;
+    }
+    100% {
+        transform: translateY(-50%) scale(1) translateX(0);
+        opacity: 1;
+    }
+}
+
+@keyframes speechBounceLeft {
+    0% {
+        transform: translateY(-50%) scale(0.8) translateX(10px);
+        opacity: 0;
+    }
+    50% {
+        transform: translateY(-50%) scale(1.05) translateX(-2px);
+        opacity: 0.8;
+    }
+    100% {
+        transform: translateY(-50%) scale(1) translateX(0);
+        opacity: 1;
+    }
+}
+
+/* Quest Log Container Queries */
+.quest-log-container {
+    container-type: inline-size;
+    container-name: quest-layout;
+}
+
+/* Default styles (mobile-first) */
+.quest-main-content {
+    width: 100%;
+    display: flex;
+    flex-direction: column-reverse;
+    gap: 1.5rem;
+}
+
+.quest-details {
+    width: 100%;
+}
+
+.quest-sidebar {
+    width: 100%;
+    padding-top: 1rem;
```

Comment:
I'd remove some of the padding so that Max aligns with the top of the content or at the very least is a little higher up
<img width="409" height="211" alt="Screenshot 2025-07-29 at 10 49 30" src="https://github.com/user-attachments/assets/f0dce0ba-e2cd-412a-9b6e-5da5b50fbea7" />